src/source/internal.rs

```diff
@@ -1,3 +1,5 @@
+extern crate mio;
```

Comment:
Marker to remove redundant extern crate.

Reply:
Removed.